src/components/payments-network/payments-network.styles.js

```diff
@@ -0,0 +1,136 @@
+import { createUseStyles } from 'react-jss'
+
+const usePaymentsNetworkStyles = createUseStyles(theme => ({
+  section: {
+    padding: '96px 72px',
+    [theme.breakpoints.sm]: {
+      padding: '96px 72px'
+    }
+  },
+  section1: {
+    paddingTop: '86px',
+    backgroundColor: '#f6f7fa',
+    display: 'flex',
+    '& > div': {
```

Comment:
I would use a dedicated className instead of targeting nested elements. The same applies for all of this cases in this file.

Reply:
These will be changed in the following PR.